tests/e2e/storage_policy_utils.go

```diff
@@ -148,3 +148,135 @@ func getDsMoRefFromURL(ctx context.Context, dsURL string) vim25types.ManagedObje
 	gomega.Expect(ds).NotTo(gomega.BeNil(), "Could not find MoRef for ds URL %v", dsURL)
 	return ds.Reference()
 }
+
+//createVsanDStoragePolicy create a vsand storage policy with given volume allocation type and category/tag map
+func createVsanDStoragePolicy(ctx context.Context, pbmClient *pbm.Client, allocationType string,
```

Comment:
There is lot of code in this PR for vsand datastore. Since it is blocked for now, either delete/comment vsand related code.